<launch>
  <node pkg="loam_velodyne" type="scanRegistration_ed" name="scanRegistration_ed" output="screen"/>
  <node pkg="loam_velodyne" type="laserOdometry_ed" name="laserOdometry_ed" output="screen"/>
  <node pkg="loam_velodyne" type="laserMapping_ed" name="laserMapping_ed" output="screen"/>
  <node pkg="loam_velodyne" type="transformMaintenance_ed" name="transformMaintenance_ed" output="screen">
    <param name="my_pose_enable" type="bool" value="false" />
  </node>

  
  <!-- ed: poseInitializerVelo 사용 -->
  <node pkg="loam_velodyne" type="poseInitializer_ed" name="poseInitializer_ed" output="screen">
    <param name="enable_init_map" type="bool" value="true" />
    <param name="map_file" type="string" value="edward.pcd" /> <!-- change here ! /-->
    <param name="init_map_record_time" type="double" value="1.0" />
    <param name="file_debug" type="bool" value="true" />
  </node>

  <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find loam_velodyne)/loam_velodyne_ed.rviz"/>

</launch>
